modules/150-user-authn/images/basic-auth-proxy/werf.inc.yaml

```diff
@@ -0,0 +1,49 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/distroless
+import:
+  - artifact: {{ $.ModuleName }}/{{ $.ImageName }}-basic-auth-proxy-artifact
+    add: /src/basic-auth-proxy
+    to: /basic-auth-proxy
+    before: setup
+docker:
+  ENTRYPOINT: [ "/basic-auth-proxy" ]
+---
+image: {{ .ModuleName }}/{{ .ImageName }}-basic-auth-proxy-src-artifact
+fromImage: common/src-artifact
+final: false
+git:
+- add: /{{ $.ModulePath }}modules/{{ $.ModulePriority }}-{{ $.ModuleName }}/images/{{ $.ImageName }}/app
+  to: /src
+  includePaths:
+  - '**/*.go'
+  - '**/go.mod'
+  - '**/go.sum'
+  stageDependencies:
+    install:
+    - '**/*.go'
+    - '**/go.mod'
+    - '**/go.sum'
+shell:
+  install:
+    - cd /src
+    - rm -rf .git
```

Comment:
```suggestion
```

You do not need this section, because you cannot download anything via git.

Reply:
thx, fixed